benchmark/single-source/DictionaryCopy.swift

```diff
@@ -17,66 +17,54 @@
 
 import TestsUtils
 
-public let DictionaryCopy = [
-  BenchmarkInfo(name: "DictionaryCopy", runFunction: run_DictionaryCopy,
-    tags: [.validation, .api, .Dictionary]),
-  BenchmarkInfo(name: "DictionaryFilter", runFunction: run_DictionaryFilter,
-    tags: [.validation, .api, .Dictionary]),
-]
+let t: [BenchmarkCategory] = [.validation, .api, .Dictionary]
```

Comment:
Nit: Please expand this name to `tags` or somesuch -- we generally dislike single-character names in the codebase.

Reply:
I have been using `t` for reducing this useless boilerplate throughout many of my past Janitor Duty PRs and Erik never objected… Do you insist on this change?

I'm also planning to propose removing the tags altogether and keeping only the `skip`, as it seems like an over-engineered aspect of out benchmark suite. Do you know if anybody uses these?

Reply:
@lorentey Does the above argument persuade you? Or is it a blocker?

Reply:
It's a nitpick. I didn't know this is a pattern now -- we can fix it later.